src/main/java/org/jenkinsci/test/acceptance/plugins/git/GitRepo.java

```diff
@@ -198,7 +204,7 @@ public void commit(final String message) {
 
     public void touch(final String fileName) {
         try {
-            FileUtils.writeStringToFile(file(fileName), "", StandardCharsets.UTF_8);
+            FileUtils.writeStringToFile(file(fileName), "", Charset.defaultCharset());
```

Comment:
although both produce the same output (it is an empty string after all) - expecting this to be used by native tooling to use the native charset.